src/parser/parser.yy

```diff
@@ -658,6 +658,9 @@ expression
     | reduce_expression {
         $$ = $1;
     }
+    | KW_VERTEX {
```

Comment:
`vertex` is a plain identifier means the default output of fetch vertex instead of keyword.